### DISCUSSION TITLE
yolov8 1280 models

### DISCUSSION THREAD

**Discussion by zwfcrazy** [ultralytics/ultralytics#1193] [General]
I notice in the current version of yolov8, only yolov8x6 has an input size of 1280. Why other models have no 1280 conterparts? Is it because it is not necessary, 640 is good enough? or simply because you haven't released those 1280 models yet?

**Comment by agentmorris** [comment]
The developers indicate [here](https://github.com/ultralytics/ultralytics/issues/338#issuecomment-1381985942) that they are planning to release a 1280px version of YOLOv8, and just hadn't had a chance to release it yet.  The last update on that issue was 1/2023.  If the devs have any updates, I'm excited to try out the 1280px v8 model!

**Reply by glenn-jocher** [reply]
@agentmorris hello! Thanks for your interest in YOLOv8 and its various model sizes. The choice of input resolution, such as 1280 for the YOLOv8x6, is a balance between performance and computational efficiency. While larger input sizes can improve accuracy, especially for detecting small objects, they also require more computational resources.

The availability of different model resolutions is based on what we've found to be the most useful and efficient during development and testing. It's not necessarily that 640 is always good enough, but it provides a good trade-off for many use cases. We continuously evaluate the performance of our models and may release additional resolutions if there's a clear benefit to the community.

As for updates on the release of a 1280px version of YOLOv8, we aim to keep the community informed through our GitHub repository. Please keep an eye on the repo for any announcements or updates regarding new model releases. Your enthusiasm for trying out new models is much appreciated, and we're excited to see how they can be applied to various tasks! 🚀

For more detailed information on model training, validation, and other modes, please refer to our documentation at https://docs.ultralytics.com.
